pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -78,6 +78,12 @@ const (
 	vgpuEnvPrefix              = "VGPU_PASSTHROUGH_DEVICES"
 	PCI_RESOURCE_PREFIX        = "PCI_RESOURCE"
 	MDEV_RESOURCE_PREFIX       = "MDEV_PCI_RESOURCE"
+	EFICode                    = "OVMF_CODE.fd"
```

Comment:
The efi detection is a nice isolated functionality now. Would you mind moving the code to a new file and a new package next to `manager.go`, like `efi/efi.go`? This file is really huge already.

Reply:
Nice idea, I like it! will do.